executor/runtime/docker/capabilities_linux.go

```diff
@@ -65,9 +83,22 @@ func setupAdditionalCapabilities(c runtimeTypes.Container, hostCfg *container.Ho
 		seccompProfile = "fuse-container.json"
 	}
 
+	// the image building apparmor+seccomp profiles are a strict superset
+	// of the fuse ones, but fuse allows access to /dev/fuse, so these need
+	// to come after fuse in the event that people asked for both
+	// fuse+image building
+	if hasCapability(containerCapabilities, titusAPI.ContainerCapability_ContainerCapabilityImageBuilding) {
+		if _, ok := addedCapabilities[SYS_ADMIN]; !ok {
+			hostCfg.CapAdd = append(hostCfg.CapAdd, SYS_ADMIN)
+		}
+
+		seccompProfile = "image-building.json"
+		apparmorProfile = "docker_image_building"
+	}
+
 	if c.SeccompAgentEnabledForPerfSyscalls() {
```

Comment:
We'll fix this soon.
Now that we have *much* better way of wrangling our seccomp profiles, this won't be so hard anymore to enable in all of them by default. (in another PR)